targets/frontend/src/pages/informations/index.js

```diff
@@ -0,0 +1,190 @@
+/** @jsx jsx  */
+
+import { SOURCES } from "@socialgouv/cdtn-sources";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { useEffect } from "react";
+import { Layout } from "src/components/layout/auth.layout";
+import { Inline } from "src/components/layout/Inline";
+import { Pagination } from "src/components/pagination";
+import { withCustomUrqlClient } from "src/hoc/CustomUrqlClient";
+import { withUserProvider } from "src/hoc/UserProvider";
+import { useDebouncedState } from "src/hooks/index";
+import { Box, Card, Input, jsx, Label, Message, Select } from "theme-ui";
+import { useQuery } from "urql";
+
+const searchEditorialContentQuery = `
+query documents($search: String!, $offset: Int = 0, $limit: Int = 50) {
+  documents(
+    where: {
+      _and: {
+        source: {_eq: "${SOURCES.EDITORIAL_CONTENT}"}
+        title: {_ilike: $search }
+      }
+    }
+    offset: $offset,
+    limit: $limit,
+    order_by:[{source: asc}, {slug: asc}]
+  )
+  {
+    id:initial_id
+    cdtnId:cdtn_id
+    title
+  }
+
+	documents_aggregate(
+    where: {
+      _and: {
+        source: {_eq: "${SOURCES.EDITORIAL_CONTENT}"},
+        title: {_ilike: $search}
+      }
+    }
+  )
+  {
+    aggregate { count }
+  }
+}
+`;
+
+export function InformationList() {
+  const router = useRouter();
+
+  const [facets, setFacets, setDebouncedFacet] = useDebouncedState(
+    {
+      itemsPerPage: Number.parseInt(router.query.itemsPerPage) || 25,
+      page: Number.parseInt(router.query.page) || 0,
+      q: router.query.q || "",
+    },
+    400
+  );
+
+  useEffect(() => {
+    router.push({ pathname: router.route, query: facets });
```

Comment:
with the new pagination, we only need to push the state to the url on every facet change and that's it